src/query/ast/src/parser/statement.rs

```diff
@@ -879,7 +879,7 @@ pub fn statement(i: Input) -> IResult<StatementMsg> {
     let show_roles = value(Statement::ShowRoles, rule! { SHOW ~ ROLES });
     let create_role = map(
         rule! {
-            CREATE ~ ROLE ~ ( IF ~ NOT ~ EXISTS )? ~ #literal_string
+            CREATE ~ ROLE ~ ( IF ~ NOT ~ EXISTS )? ~ #parameter_to_string
```

Comment:
all `ROLE` related opeartions should be `parameter_to_string`, such as `DROP ROLE`...

Reply:
![image](https://github.com/datafuselabs/databend/assets/33082201/b579b12e-8364-4e35-912d-bf22b3deda05)

Yes. drop/create role, grant/revoke role, set role,

Reply:
the easy way, you need to all `ROLE ~ #literal_string`.